packages/cadl-ranch/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @azure-tools/cadl-ranch
 
+## 0.13.5
+
+### Patch Changes
+
+- 1cdf0c8: updated dependencies @azure-tools/cadl-ranch-api@0.4.4
```

Comment:
It is better to follow https://github.com/Azure/cadl-ranch?tab=readme-ov-file#release-a-new-version to make the bump version PR so that all changes are included.

Reply:
there's no code change compare to latest main